ext/bcmath/bcmath.stub.php

```diff
@@ -2,22 +2,22 @@
 
 /** @generate-function-entries */
 
-function bcadd(string $left_operand, string $right_operand, ?int $scale = null): string {}
+function bcadd(string $num1, string $num2, ?int $scale = null): string {}
 
-function bcsub(string $left_operand, string $right_operand, ?int $scale = null): string {}
+function bcsub(string $num1, string $num2, ?int $scale = null): string {}
 
-function bcmul(string $left_operand, string $right_operand, ?int $scale = null): string {}
+function bcmul(string $num1, string $num2, ?int $scale = null): string {}
 
-function bcdiv(string $dividend, string $divisor, ?int $scale = null): string {}
+function bcdiv(string $num1, string $num2, ?int $scale = null): string {}
 
-function bcmod(string $dividend, string $divisor, ?int $scale = null): string {}
+function bcmod(string $num1, string $num2, ?int $scale = null): string {}
 
-function bcpowmod(string $base, string $exponent, string $modulus, ?int $scale = null): string {}
+function bcpowmod(string $base, string $exp, string $mod, ?int $scale = null): string {}
```

Comment:
TBH I liked `$exponent` and `$modulus` a bit more.

Reply:
Okay, I've restored $exponend/$modulus here and used them on the GMP side as well.